template-search-extension/src/config.js

```diff
@@ -78,7 +78,7 @@ const whitelist = [];
 // do not change the following unless you know what you are doing
 const config = {
     availability,
-    category: 'search',
+    category: 'widget',
```

Comment:
I'm sad to see this :cry: But the current implementation of the Playground force you to set the category as widget. We need to update this @germainruget and/or @LaurentMox